client/asset/driver.go

```diff
@@ -154,6 +154,16 @@ func DecodeCoinID(assetID uint32, coinID []byte) (cid string, err error) {
 		tkn.ParentID, dex.BipIDSymbol(tkn.ParentID), assetID, dex.BipIDSymbol(assetID))
 }
 
+// IsToken checks if the asset ID is for a token and returns the token's parent
+// ID.
+func IsToken(assetID uint32) (is bool, parentID uint32) {
+	token, is := tokens[assetID]
+	if !is {
+		return
+	}
+	return true, token.ParentID
+}
```

Comment:
Will you please move this down next to `func TokenInfo` around line 230?

Also, I think you need to follow the pattern in that function of using `driversMtx` to access the `tokens` map.

Although I think you should just use `asset.TokenInfo`.  See my other comment in core.go.